core/platform/storage/cloud_storage_services.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
```

Comment:
Shouldn't this file have an associated test file?

Reply:
Yes, it should.

Reply:
Done.